src/main/scala/cromwell/engine/EngineFunctions.scala

```diff
@@ -0,0 +1,28 @@
+package cromwell.engine
+
+import cromwell.binding.IoInterface
+import cromwell.binding.expression.WdlStandardLibraryFunctions
+import cromwell.binding.values._
+
+import scala.language.postfixOps
+import scala.util.{Success, Try}
+
+/**
+ * Default implementation of Wdl Standard Library functions executed at the workflow level.
+ * This class is abstract to enforce that Backends extend it and customize the behavior if needed.
+ */
+abstract class WorkflowEngineFunctions(override val interface: IoInterface, val context: WorkflowContext) extends WdlStandardLibraryFunctions {
+
+  override def tempFilePath: String = context.root
+  override def globPath(glob: String): String = context.root
```

Comment:
I see.  Thanks for the detailed explanation, I forgot to account for the fact that JES has to delocalize the glob files into its own directory